src/Microsoft.VisualStudio.LanguageServices.Razor/ProjectSystem/RazorProjectHost.cs

```diff
@@ -170,6 +187,25 @@ protected override Task InitializeCoreAsync(CancellationToken cancellationToken)
             });
         }
 
+        private async Task<string> DetectFallbackLanguageVersion()
+        {
+            var references = await _commonServices.ActiveConfiguredProjectAssemblyReferences.GetResolvedReferencesAsync().ConfigureAwait(false);
+            foreach (var reference in references)
+            {
+                var fullPath = await reference.GetFullPathAsync().ConfigureAwait(false);
+                if (fullPath.EndsWith(MvcAssemblyFileName, StringComparison.OrdinalIgnoreCase))
+                {
+                    var assemblyName = await reference.GetAssemblyNameAsync().ConfigureAwait(false);
```

Comment:
What happens when a reference is unresolved? Does it return null?

Reply:
In that case the `reference` itself wouldn't be included in the collection.